Tests/SonarScanner.MSBuild.Common.Test/ILoggerTests.cs

```diff
@@ -338,13 +379,13 @@ public void ILogger_Log_InfoAndDebug()
     }
 
     [TestMethod]
-    public void ILogger_Log_UnknownLogVerbosity()
+    public void ILogger_Log_UIWarning()
     {
         var logger = Substitute.For<ILogger>();
-        logger.Invoking(x => x.Log((LoggerVerbosity)100, "message")).Should().ThrowExactly<ArgumentOutOfRangeException>().WithMessage("""
-            Unsupported log level.
-            Parameter name: level
-            Actual value was 100.
-            """);
+        logger.Log(LoggerVerbosity.Info, "info message");
+        logger.Log(LoggerVerbosity.Debug, "debug message");
+        logger.ReceivedCalls().Should().HaveCount(2);
+        logger.Received(1).LogInfo("info message");
+        logger.Received(1).LogDebug("debug message");
```

Comment:
I'm not sure I understand what we achieve by having this test.

Reply:
Good catch, it was a copy-paste leftover from when I started implementing tests for the ConsoleLogger, I'll remove it